package.json

```
@@ -52,6 +52,11 @@
     ],
     "commands": [
       {
+        "command": "onevscode.fileTreeView-open",
+        "title": "Set model directory",
+        "category": "ONE",
+        "_comment": "Call this command to set the model directory of local machine."
+      },      {
```

You need a new line.

```suggestion
      },
      { 
```

---

Fixed. Thanks.